java/openjdk11/Portfile

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem       1.0
+
+name             openjdk11
+version          11
+
+categories       java devel
+maintainers      {breun.nl:nils @breun} openmaintainer
+platforms        darwin
+license          GPL-2
+supported_archs  x86_64
+
+description      Open Java Development Kit 11
+
+long_description Production-ready, free and open-source build of the Java \
+                 Development Kit, an implementation of the Java Standard \
+                 Edition (SE) 11 Platform. OpenJDK is the official reference \
+                 implementation of Java SE. Included components are the \
+                 HotSpot virtual machine, the Java class library and the Java \
+                 compiler. 
+
+homepage         https://jdk.java.net/11/
+master_sites     https://download.java.net/java/GA/jdk11/28/GPL/
+distname         openjdk-${version}+28_osx-x64_bin
```

Comment:
Isn't it likely that there will be future versions of JDK 11? I think it would be prudent to parameterize the `28`. The port version should probably be `11_28`.

Reply:
I actually was about to submit my own port for openjdk11 before I saw this PR; see here for how I thought it should be done:
https://github.com/amake/macports-ports/commit/641e9319ae946129cbc7c5a8fce7e726697dccd4

Reply:
The 28 is only part of the build number. This is the output of `java -version`:

```
openjdk version "11" 2018-09-25
OpenJDK Runtime Environment 18.9 (build 11+28)
OpenJDK 64-Bit Server VM 18.9 (build 11+28, mixed mode)
```

The next version of OpenJDK will probably be 11.0.1.

Reply:
I've introduced a `build` variable for the 28.